### ISSUE TITLE
Insecure demo file disclosure

### ISSUE THREAD

**Issue by Xyntax** [zikula/core#3260]
Unauthorized file manipulation - `/vendor/vakata/jstree/demo/filebrowser/index.php`

http://zikula.org/vendor/vakata/jstree/demo/filebrowser/

Possible unauthorized SSRF - `/javascript/js-webshim/dev/shims/FlashCanvasPro/proxy.php`
```
if (extension_loaded('curl')) {
    // Use cURL extension
    $ch = curl_init($url);
//  curl_setopt($ch, CURLOPT_FOLLOWLOCATION, true);
//  curl_setopt($ch, CURLOPT_MAXREDIRS, 10);
    curl_exec($ch);
    curl_close($ch);
} else {
    // Use the http:// wrapper
    readfile($url);
}
```

Please make a review.

**Comment by Guite** [comment]
Maybe we should report that at the vendor projects so they can provide a central fix?

**Comment by Xyntax** [comment]
Yes we can report that, but I'm afraid not all vendors cares about it. 
Maybe we should make a simple review ourselves.

**Comment by Guite** [comment]
We could possibly block access to these files using the `.htaccess` file in the root folder.

**Comment by Guite** [comment]
`javascript/js-webshim/dev/shims/FlashCanvasPro/proxy.php` is already blocked by `javascript/.htaccess`.

**Comment by Guite** [comment]
Added a fix for the first one in #3261

**Comment by Guite** [comment]
Reported to vendor here: https://github.com/vakata/jstree/issues/1651
